CustomScript/customscript.py

```diff
@@ -93,7 +93,7 @@ def dummy_command(operation, status, msg):
 
 
 def parse_context(operation):
-    hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName, console_logger=waagent.LogToCon, file_logger=waagent.LogToFile)
+    hutil = Util.HandlerUtility(waagent.Log, waagent.Error, ExtensionShortName, console_logger=waagent.LogToConsole, file_logger=waagent.LogToFile)
     hutil.do_parse_context(operation)
```

Comment:
LogToConsole != WriteToConsole here.

Reply:
And same for LogToFile

Reply:
What do you mean? Sorry I don't get your comment...